internal/idp/providers/oauth/oauth2.go

```diff
@@ -99,6 +99,15 @@ func (p *Provider) BeginAuth(ctx context.Context, state string, params ...idp.Pa
 	if !loginHintSet {
 		opts = append(opts, rp.WithPrompt(oidc.PromptSelectAccount))
 	}
+
+	if p.RelyingParty.IsPKCE() {
+		codeChallenge := oidc.NewSHACodeChallenge(idp.CodeVerifier())
```

Comment:
the code_verifier needs to be stored to be sent on the code exchange
also need to look into how that can be done best in our case (using the cookie handler)

Reply:
@livio-a any hint on how I could store the code_verifier?